backend/src/main/java/com/github/mariakohnen/packbag/backend/service/PackingListService.java

```diff
@@ -79,5 +79,17 @@ public PackingItem generateNewItem(CreatePackingItemDto createPackingItemDto) {
                 .name(createPackingItemDto.getName())
                 .build();
     }
+
+    public PackingList deleteItemFromPackingList(String id, String itemId) {
+        PackingList updatedPackingList = packingListRepository.findById(id)
+                .orElseThrow(() -> new NoSuchElementException("Packing list with id: " + id + " was not found!"));
+        List<PackingItem> actualItemList = updatedPackingList.getPackingItemList();
+        if (actualItemList != null) {
+            List<PackingItem> updatedItemList = new ArrayList<>(actualItemList);
```

Comment:
So ganz verstehe ich nicht, warum du an dieser Stelle eine new ArrayList erstellst. 
Würde nicht auch so etwas ähnliches wie 
List\<PackingItem\> updatedItemList = actualItemList.removeIf...
gehen?
Aber ich war solange nicht im Backend unterwegs, bin mir da auch nicht sicher...

Reply:
ja ich verstehe was du meinst. Habe es gerade auch nochmal ausprobiert. Das Problem ist, dass die items in einer ArrayList gespeichert werden. Und diese darf man nicht verändern, von der Länge her zumindest. Ansonsten wirft er eine UnsupportedOperationException. Daher muss man dann eine neue Liste erstellen... Aber bestimmt gibt es eine elegantere Lösung.